react/src/demo/example-data/terrain-worker.js

```diff
@@ -0,0 +1,1254 @@
+(() => {
```

Comment:
Is this file needed?

Reply:
Yes I think so. As an example of a not-inlined but externally served worker script